data/pages-content/faq.md

```diff
@@ -41,7 +54,7 @@ We are always keen to hear new ideas on how to enrich the programme and other ac
 
 ## **Q. When do you accept talk proposals?**
 
-A. The Call for Proposals (CFP) will be launched in early March. We are thrilled to receive proposals on any topic that could be of interest to our community.
+A. The Call for Proposals (CFP) will run for two weeks between 6-19 March 2023. We are thrilled to receive proposals on any topic that could be of interest to our community.
 
 Especially if you are considering becoming a speaker for the first time, or if you are from an underrepresented group, we strongly encourage you to send in your proposals. There will be a speaker mentorship programme as well as broader support options. Make sure to subscribe to our [EuroPython Blog](https://blog.europython.eu/#/portal/signup) for updates.
```

Comment:
> There will be a speaker mentorship programme as well as broader support options

Can we be more explicit here: `We will run a speaker mentorship programme this year...`